packages/next/src/server/lib/patch-fetch.ts

```diff
@@ -387,7 +388,7 @@ export function patchFetch({
             revalidate = 0
             cacheReason = 'fetchCache = default-no-store'
           } else {
-            cacheReason = 'auto cache'
+            cacheReason = isUsingNoStore ? 'noStore call' : 'auto cache'
```

Comment:
Should this be it's own condition so `revalidate` is also explicitly set to `0`?